### DISCUSSION TITLE
Kreditkarte wird nicht akzeptiert

### DISCUSSION THREAD

**Discussion by Franzl2406** [aclap-dev/video-downloadhelper#856] [Help and Q&A]
### Your question

gibt es weitere Möglichkeiten der Bezahlung?

### Addon & CoApp version and details

version: 9.0.2.3
channel: stable
build date: 2024-05-27
build options: linuxlic: false, noyt: false, browser: firefox
lang: de
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:126.0) Gecko/20100101 Firefox/126.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\Franz",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by jcfrog** [comment]
Hi,
Sorry you have to use Stripe platform.
